core/services/streams/delegate.go

```diff
@@ -43,11 +43,7 @@ func (d *Delegate) BeforeJobDeleted(jb job.Job)                {}
 func (d *Delegate) OnDeleteJob(context.Context, job.Job) error { return nil }
 
 func (d *Delegate) ServicesForSpec(ctx context.Context, jb job.Job) (services []job.ServiceCtx, err error) {
-	if jb.StreamID == nil {
-		return nil, errors.New("streamID is required to be present for stream specs")
-	}
-	id := *jb.StreamID
-	lggr := d.lggr.Named(fmt.Sprintf("%d", id)).With("streamID", id)
+	lggr := d.lggr.Named(fmt.Sprintf("job-%d", jb.ID)).With("jobID", jb.ID)
```

Comment:
These are typically capitalized:
```suggestion
	lggr := d.lggr.Named(fmt.Sprintf("Job-%d", jb.ID)).With("jobID", jb.ID)
```
And would it make sense to dot separate it? :shrug: 
```suggestion
	lggr := d.lggr.Named(fmt.Sprintf("Job.%d", jb.ID)).With("jobID", jb.ID)
```